runtime/j9vm/jvm.c

```diff
@@ -3991,10 +4002,30 @@ JVM_LoadLibrary(const char *libName, jboolean throwOnFailure)
 		JavaVM *vm = (JavaVM *)javaVM;
 		(*vm)->GetEnv(vm, (void **)&env, JNI_VERSION_1_2);
 		if (NULL != env) {
-			char errMsg[512];
-			jio_snprintf(errMsg, sizeof(errMsg), "Failed to load library \"%s\"", libName);
-			errMsg[sizeof(errMsg) - 1] = '\0';
+			char errBuf[512];
+			char *errMsg = errBuf;
+			int bufSize = sizeof(errBuf);
+			const char *portMsg =
+#if defined(WIN32)
+			 !attemptedLoad ? "" :
+#endif /* defined(WIN32) */
+			j9error_last_error_message();
```

Comment:
Lines 4010 and 4012 should be indented one tab (and no spaces) more than line 4008.